.github/CODEOWNERS

```diff
@@ -1,23 +1,34 @@
-/distsql @pingcap/exec-reviewers
-/executor @pingcap/exec-reviewers
-/expression @pingcap/exec-reviewers
-/types @pingcap/exec-reviewers
-/util/chunk @pingcap/exec-reviewers
-/util/disk @pingcap/exec-reviewers
-/util/execdetails @pingcap/exec-reviewers
-/util/expensivequery @pingcap/exec-reviewers
-/util/filesort @pingcap/exec-reviewers
-/util/memory @pingcap/exec-reviewers
-/util/sqlexec @pingcap/exec-reviewers
-
-/planner @pingcap/planner-reviewers
-/statistics @pingcap/planner-reviewers
-/util/ranger @pingcap/planner-reviewers
-/util/plancodec @pingcap/planner-reviewers
-/bindinfo @pingcap/planner-reviewers
-
-/ddl @pingcap/co-ddl
-/domain @pingcap/co-ddl
-/infoschema @pingcap/co-ddl
-/meta @pingcap/co-ddl
-/owner @pingcap/co-ddl
+/bindinfo @rebelice
+/cmd @eurekaka @rebelice
+/config @tiancaiamao @qw4990
+/ddl @zimulala @tiancaiamao
+/distsql @crazycs520 @qw4990
+/docs @zz-jason @bb7133
+/domain @lysu @xhebox
+/errno @morgo @tangenta
+/executor @lysu @XuHuaiyu @wshwsh12
+/expression @XuHuaiyu @wjhuang2016
+/infoschema @djshow832 @lysu
+/kv @disksing @AndreMouche
+/lock @crazycs520
+/meta @tangenta @AilinKid
+/metrics @jackysp @lzmhhh123
+/owner @zimulala @wjhuang2016
+/planner @winoros @rebelice @hanfei1991 
+/plugin @lysu
+/privilege @morgo
+/server @coocood @morgo
+/session @cfzjywxk @tiancaiamao
+/sessionctx @qw4990
+/sessionctx/variable @morgo
+/statistics @winoros @eurekaka
+/store @disksing @AndreMouche @MyonKeminta
+/structure @zimulala
+/table @wjhuang2016
+/tablecodec @tangenta
+/telemetry @breeswish @lzmhhh123 @kaixu120811
+/tests @ichn-hu
```

Comment:
why am I the code owner of `/tests`?

Reply:
It is based on 

```
$ find -EL tests -type f -regex '.*\.(md|go)' | xargs -n 1 git blame --porcelain | grep  "^author " | sort | uniq -c | sort -nr | head -10
   3 author Ping Yu
   2 author 虎
   2 author jianzhiyao
   2 author Tjianke
   2 author Jack Yu
   1 author Zhou Kunqin
   1 author Yuanjia Zhang
   1 author Feng Liyuan
```

but yes. We might think of the real problem on auto assign. Possibly make use of https://book.prow.tidb.io/#/plugins/blunderbuss cc @hi-rustin @Mini256